docs/compression_algorithms/CompressWeights.md

```diff
@@ -61,6 +61,64 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
 compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM, ratio=0.8, dataset=nncf_dataset) # model is openvino.Model object
 ```
 
+- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
+original precision and 4-bit.
+Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ.
+It requires to set `awq` to `True` additionally to data-based mixed-precision algorithm.
```

Comment:
The table with data-free vs data-aware mode should be updated. 
Customers will definitely ask how it improves accuracy and how long does it take in comparison with data-free and data-aware without AWQ. 
Since experiments show benefits of both methods - data-aware with AWQ and data-aware without AWQ, I'd show accuracy for these 2 modes.